### ISSUE TITLE
Create a webform to create PRs

### ISSUE THREAD

**Issue by dns2utf8** [spacedirectory/meta#3]
The website would present a form for new entries.
Once the new entry is validated and checked against duplicates the users request will be translated into a pull request against the current master of the database repo.

**Comment by dbrgn** [comment]
We could also offer a dropdown of existing entries to be updated.

**Comment by romedius** [comment]
concerning the updates:
there is an issue_mail field in the spaceapi 1.13 specification.
We could mail a validation token to the given adress to allow updating the URL changes.

**Comment by dbrgn** [comment]
Yes, but what if someone entered a wrong e-mail once and cannot update it anymore?

And if we simply believe someone saying "I cannot access the original e-mail anymore", then what additional security does that validation token bring us?

I think having no authentication but peer review on Github is probably the more pragmatic approach.

**Comment by romedius** [comment]
@dbrgn the original email adress is in the json file provided from your server. So you should be able to change it. As long as you own the Domain, it is possible.
It would be an extra solution. For the tiny dataset we curate now, peer reviewed Pull requests are enouth, you are right :)

**Comment by dns2utf8** [comment]
Something like this?

1. User submits URL
2. Server fetches the URL
3. Server sends an eMail to the listed address
4. User clicks the token received by eMail
5. Server creates PR with entry

**Comment by dbrgn** [comment]
:+1: 

I'd suggest implementing the form with direct PR creation first though, and only adding verification once we start having problems with that approach.

**Comment by rorist** [comment]
+1 for the webform and direct PR, anybody in a hackerspace should be able to create and update the url, not only the one controlling the email/server. People will probably loss acces to the email, then we'd need a mechanism to re-authenticate the space url, which seems complicated.
